.github/workflows/vulnerabilities.yml

```diff
@@ -0,0 +1,52 @@
+# Analyze for common vulnerabilities using GitHub's CodeQL
+#
+name: 'Vulnerability Scanning'
+
+on:
+    - pull_request
```

Comment:
I think doing this at a PR level is kind of bad.

Say some new vulnerability comes out. Now suddenly all PRs start failing because of it. Once we have fixed it _in another PR_ we need to then merge in master for every single open PR to get it green.

A better model IMO would be to either run this on master periodically and e.g. create issues. WDYT?

Reply:
Yup agreed, I'm also not sure this is the best way to scan for vulnerabilities. For now, I'll enable image scanning on Docker Hub & ECR, we can keep the original issue open to explore further how best to address this.